tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -583,7 +583,9 @@ func (az *AzureProvider) constructOSProfile(ctx context.Context) (osProfile *com
 	data := `$url = "https://raw.githubusercontent.com/ansible/ansible/devel/examples/scripts/ConfigureRemotingForAnsible.ps1"
         $file = "$env:temp\ConfigureRemotingForAnsible.ps1"
         (New-Object -TypeName System.Net.WebClient).DownloadFile($url,  $file)
-        powershell.exe -ExecutionPolicy ByPass -File $file`
+        powershell.exe -ExecutionPolicy ByPass -File $file
+        powershell -NonInteractive -ExecutionPolicy Bypass New-NetFirewallRule -DisplayName "EnableConsoleLogs"
+        -Direction Inbound -Action Allow -Protocol TCP -LocalPort 10250 -EdgeTraversalPolicy Allow`
```

Comment:
Would there be an issue with a new line being introduced here?

Reply:
nope, it will still work.